tests/copulas/multivariate/test_gaussian_copula.py

```diff
@@ -22,3 +23,19 @@ def test_deprecation_warnings(self):
             # Check
             assert len(warns) == 0
             assert len(result) == 10
+
+    def test_sample(self):
+        copula = GaussianMultivariate()
+        data = pd.DataFrame([[-1, 0, 1], [1, -1, 0], [0, 1, -1]])
+        copula.fit(data)
+
+        # Run
+        result = copula.sample(1000000)
+
+        # Check
+        assert len(result) == 1000000
```

Comment:
Change this assertion to:
```python
assert result.shape == (1000000, 3)
```
This is because, the `for` loop below won't raise any error if `error.shape[1] < data.shape[1]`